src/DssSpell.sol

```diff
@@ -15,23 +15,33 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 pragma solidity 0.6.12;
-
-// Enable ABIEncoderV2 when onboarding collateral
+// Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
 // pragma experimental ABIEncoderV2;
+
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./DssSpellCollateral.sol";
 
-contract DssSpellAction is DssAction, DssSpellCollateralAction {
+interface ERC20Like {
+    function approve(address, uint256) external returns (bool);
+}
 
+interface RwaUrnLike {
+    function lock(uint256) external;
+    function draw(uint256) external;
+}
+
+contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/0114d88f41cd3cf49ba818d2b3c4f159490a6944/governance/votes/Executive%20vote%20-%20July%2013%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/master/governance/votes/Executive%20vote%20-%20July%2029%2C%202022.md -q -O - 2>/dev/null)"
```

Comment:
```suggestion
    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/969f04cfec25e56791fbe4503bcbe2df7a58df1e/governance/votes/Executive%20vote%20-%20July%2029%2C%202022.md -q -O - 2>/dev/null)"
```

Reply:
Once you make this change, you have an approval from me to deploy.

Reply:
Resolving as it is fixed.